docs/usage/executor/helm.md

```diff
@@ -14,6 +14,14 @@ To execute `helm` commands, send message in the following format in the channel
 
 At least one `helm` executor needs to be enabled and bound to a given channel.
 
+:::caution
+Using the interactive filter input field causes the Helm command to be re-executed. Be careful when using it for read-write commands.
```

Comment:
This is a good warning, but WDYT also about saying explicitly about such behavior in the filtering feature?

https://docs.botkube.io/usage/interactive-output-filtering

Reply:
good point, added 👍